config/rbac/module_loader_cluster_role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: module-loader
```

Comment:
```suggestion
  name: kmm-operator-module-loader
```

Reply:
You revealed pretty fast a pain point regarding the naming and the `kustomize` manifests, with which I would like some help :)

#### Context
We have setup the [namePrefix](https://github.com/mresvanis/rh-kernel-module-management/blob/main/config/default/kustomization.yaml#L9), so all known to kustomize resources are named with the `kmm-operator-` prefix by default. But at the same time, I didn't find a way to use this transformer for the `resourceNames` field of the `ClusterRole`.  So I had to manually put the `kmm-operator-` prefix to the `resourceNames` field of the `ClusterRole`.

#### Ideas?
As this might break the whole thing if someone chooses to replace the `namePrefix` when running `kustomize`, do you think we could maybe opt-out these manifests from the `namePrefix` transformer?

Reply:
Thanks for the help! The [solution](https://github.com/rh-ecosystem-edge/kernel-module-management/pull/53/files#diff-006ba62583b26e070f2015bafbd541d453d58f2d3136ef65522d835f8b002ce8) you pointed out works as expected.